src/jobs/index.module.css

```diff
@@ -0,0 +1,111 @@
+.omVariant {
+  padding: 0 1rem 4rem;
+  position: relative;
+  margin-bottom: 3rem;
+}
+
+@media (max-width: 500px) {
+  .omVariant {
+    margin-bottom: 1rem;
+    padding-bottom: 1rem;
+  }
+}
+
+.omVariant__title {
+  font-weight: bold;
+  margin: 1.8rem 0;
+  font-size: 3.3rem;
+
+  background: linear-gradient(
+    to right,
+    var(--color-primary),
+    var(--color-secondary1),
+    var(--color-secondary2)
+  );
+  background-size: 200% 200%;
+  animation: backgroundPos 10s ease-in-out infinite;
+  background-clip: text;
+  transition: color 0.2s ease-in-out;
+  color: rgba(0, 0, 0, 0);
+}
+.omVariant__title[data-no-animation='true'] {
+  animation: none;
+}
+
+.omVariant__subTitle {
+  font-size: 1.4rem;
+  margin: 0 0 2.3rem 1.5rem;
+}
+
+.omVariant__wrapper__p {
+  text-align: justify;
+  position: relative;
+}
+
+.omVariant__wrapper {
+  display: grid;
+  grid-template: 'a b' / 1fr auto;
+}
+
+.omVariant__blob {
+  grid-area: b;
+  float: right;
+  margin: 1rem -1rem;
+}
+@media (max-width: 700px) {
+  .omVariant__wrapper {
+    grid-template: 'a' 'b' /1fr;
+  }
+}
+
+.withSubTitle {
+  margin-bottom: -1rem;
+}
+.job__listing {
+  padding: 0 1rem 7rem;
+  position: relative;
+  margin-bottom: 3rem;
+}
+
+@media (max-width: 500px) {
+  .job__listing__container {
+    margin-bottom: 1rem;
+    padding-bottom: 1rem;
+  }
+}
+
+.job__listing__container {
+  background-color: var(--color-secondary2);
+  margin: 2rem 0;
+  border-radius: 2rem;
+  padding: 2rem;
+  display: grid;
+  grid-template: 'a b' 1fr / 1fr auto;
+}
+@media (max-width: 500px) {
+  .job__listing__container {
+    grid-template: 'a' 'b';
+    gap: 1rem;
+  }
+}
+
+.job__listing__container:nth-child(2n) {
+  background-color: var(--color-secondary3);
+}
+
+.aspect__ratio {
+  position: relative;
+  width: calc(100% - 2rem);
+  height: 0;
+  padding-bottom: calc(56.18% - 2rem);
```

Comment:
Is the percentage here an arbitrary value or a percentage for a given aspect ratio? Maybe it should have a comment if the latter.